force-app/main/default/classes/BotTest.cls

```diff
@@ -1,6 +1,16 @@
 @isTest
 public class BotTest {
     
+    static testMethod void testAutoBotController() {
```

Comment:
Method `testAutoBotController` is not annotated. Remove the `testMethod` keyword and annotate with `@IsTest` instead.